src/bootstrap.sh

```diff
@@ -9,34 +9,6 @@ if [ ! -f ./config.ini ]; then
     cp config.ini.example config.ini
 fi
 
-# Test GPS
-if [ ! -z $1 ]; then
-    echo -e "Skipping GPS test"
-else
-    echo -e "Starting GPS"
-    sudo killall -9 gpsd 2> /dev/null
-    sudo service gpsd stop
-    if [ -e /dev/ttyUSB0 ]; then
-        sudo gpsd /dev/ttyUSB0
-        echo -e "Waiting for GPS to sync"
-        TRIES=0
-        until ../experiments/gps.sh | grep -v null  > /dev/null; do
-            echo -e "GPS failed to sync."
-            sleep 1
-            TRIES=$[ $TRIES + 1 ]
-            if [ $TRIES -ge 10 ]; then
-              echo -e "GPS couldn't get a fix, giving up"
-              break;
-            fi
-        done
-        echo -e "GPS successfully got location"
-    else
-        echo -e "No GPS device found"
-        exit 1
-    fi
-fi
-
-
```

Comment:
Yeah, that makes sense. My problem is just that hard-coding the device in here will break everyone's configuration who doesn't have a GPS that shows up where yours does in udev, which is actually all of my setups as well. There's a large number of possible places they show up: /dev/ttyUSBx, /dev/ttySx, /dev/ttyACMx, /dev/ttyAMAx, etc. I don't think we should be forcing users of the application to run GPSD with parameters that only work with specific GPS devices in user-specific environments. This, for example, running on my Pi or laptop would require me (or anyone else) to make a code change every time I pulled the repo, and other applications using my system GPSD would also be disrupted. Maybe the thing to do here is augment the documentation with a link to a no-nonsense guide to configuring GPSD or doing a short write-up on identifying your USB GPS.

Reply:
Let me know what you think.

Reply:
that's a good idea, I also agree that hard coding the GPS device is a bad idea but we could put the gps device serial path in the config file pretty easily.

Reply:
I assume that if we skipped GPSD and read from teh device directly that this wouldn't disrupt other applications using gpsd right? I think it could be a lot cleaner as long as we can easily configure the gps device. All the gps stuff happens in python so it shouldn't be too hard I think.